numpy/linalg/tests/test_linalg.py

```diff
@@ -338,7 +338,7 @@ def test_sq_cases(self):
 
 class LinalgNonsquareTestCase(object):
 
-    def test_sq_cases(self):
```

Comment:
Now that they are run, they fail, because

```python
>>> import numpy as np
>>> a = np.random.rand(8, 11)
>>> b = np.random.rand(11)
>>> np.linalg.lstsq(a, b)
```

doesn't make any sense